src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculate {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```